docs/features/techdocs/creating-and-publishing.md

```diff
@@ -30,11 +30,6 @@ the documentation template.
 Create an entity from the documentation template and you will get the needed
 setup for free.
 
-!!! warning Currently the Backstage Software Templates are limited to create
```

Comment:
There's also a reference in https://github.com/spotify/backstage/blob/master/microsite/blog/2020-08-05-announcing-backstage-software-templates.md not sure if you want a blog post updated or not?

Reply:
I'd say it'd be fine to leave the blog post intact. It's a bit of a snapshot in time, but it also doesn't make clear that you can't scaffold repos on user accounts tbh.